src/utils/apiRequests/api.ts

```diff
@@ -1,18 +1,48 @@
+import type { ImpersonationData } from '../../features/user/Settings/ImpersonateUser/ImpersonateUserForm';
+
 import { getQueryString } from './getQueryString';
 import getsessionId from './getSessionId';
 import { APIError, ClientError } from '@planet-sdk/common';
 import { validateToken } from './validateToken';
-import { ImpersonationData } from '../../features/user/Settings/ImpersonateUser/ImpersonateUserForm';
 import { setHeaderForImpersonation } from './setHeader';
 
 const INVALID_TOKEN_STATUS_CODE = 498;
 
+interface GetAccountInfo {
+  tenant: string | undefined;
+  token: string | null;
+  impersonationData?: ImpersonationData;
+}
+
+interface GetAuthRequestOptions {
+  tenant?: string | undefined;
+  url: string;
+  token: string | null;
+  logoutUser: (value?: string | undefined) => void;
+  header?: Record<string, string> | null;
+  queryParams?: { [key: string]: string };
+  version?: string;
+}
+
+type GetRequestOptions = Omit<
+  GetAuthRequestOptions,
+  'header' | 'logoutUser' | 'token'
+>;
```

Comment:
I wonder if for better readability we should allow some redundancy here and use a normal `interface` definition as well to have every interface independant of each other. (One could also write a base interface with all kinds of elements and use Pick to contruct every other type or some other way to use the reuse elements from other types, but I'd prefer to have them independant from each other.)